README.md

```diff
@@ -99,7 +99,7 @@ In the third, run:
 
 ``` shell
 export GST_PLUGIN_PATH=$PWD/target/debug:$GST_PLUGIN_PATH
-gst-launch-1.0 webrtcsink name=ws videotestsrc ! ws. audiotestsrc ! ws.
+gst-launch-1.0 webrtcsink name=ws stun-server=stun://stun.l.google.com:19302 videotestsrc ! ws. audiotestsrc ! ws.
```

Comment:
No need for a stun-server here, the instructions are explicitly to run the example locally :)

Reply:
Well, "locally" is not necessarily so locally as you know ;)